src/modules/common/components/ScrollingSearchBox.js

```diff
@@ -1,26 +1,35 @@
+// @flow
+
+import type { Element } from 'react'
 import React from 'react'
-import PropTypes from 'prop-types'
 
 import SearchInput from './SearchInput'
 import Headroom from './Headroom'
 
 const SEARCH_BAR_HEIGHT = 45
 
-export class ScrollingSearchBox extends React.PureComponent {
-  static propTypes = {
-    filterText: PropTypes.string.isRequired,
-    onFilterTextChange: PropTypes.func.isRequired,
-    spaceSearch: PropTypes.bool,
-    children: PropTypes.element.isRequired,
-    placeholderText: PropTypes.string.isRequired
-  }
+type PropsType = {
+  filterText: string,
+  onFilterTextChange: (string) => void,
+  spaceSearch: boolean,
```

Comment:
This should be `?boolean` because it wasn't/isn't required, shouln't it?

Reply:
It's not required because it defaults to false.